File: Java/프로그래머스/1/12977. 소수 만들기/소수 만들기.java
class Solution {
    public boolean is_prime(int number) {
        if (number < 2) return false;
        for (int i=3; i<number; i++) {
            if (number % i == 0) return false;
        }
        return true;
    }
    
    public int solution(int[] nums) {
        int answer = 0;
        
        for (int i=0; i<nums.length; i++) {
            for (int j=i+1; j<nums.length; j++){
                for (int k=j+1; k<nums.length; k++) {
                    if (is_prime(nums[i] + nums[j] + nums[k])) answer += 1;
                }
            }
        }
        return answer;
    }
}

File: Python3/프로그래머스/1/12977. 소수 만들기/소수 만들기.py
import itertools as it

def is_prime(number):
    if number < 2 : 
        return False
    for i in range(3, number):
        if number % i == 0:
            return False
    return True

def solution(nums):
    answer = 0
    for i,j,k in it.combinations(nums, 3) :
        if is_prime(i+j+k) :
            answer += 1
    return answer